samples/subsys/power/device_pm/sample.yaml

```diff
@@ -4,3 +4,23 @@ tests:
   ospm.dev_idle_pm:
     platform_whitelist: nrf52840_pca10056 nrf52_pca10040
     tags: power
+    harness: console
+    harness_config:
+      type: multi_line
+      ordered: false
+      regex:
+        - "Device PM sample app start"
+        - "open()"
+        - "parent resuming.."
+        - "child resuming.."
+        - "Async wakeup request queued"
+        - "Dummy device resumed"
+        - "write()"
+        - "transfer()"
+        - "read()"
+        - "transfer()"
```

Comment:
duplicate lines do not work with sanitycheck matching, so this will fill due to the duplicate transfer() lines.